templates/internet-of-things/index.html

```diff
@@ -141,28 +141,53 @@ <h3>Networking</h3>
         </div>
       </div>
     </div>
+    <div class="col-6 p-tile">
+      <div class="row p-tile__row">
+        <div class="col-3 col-medium-3">
+          <div class="u-align--center">
+            {{ 
+              image (
+                url="https://assets.ubuntu.com/v1/c75046c4-Smart+City.svg",
+                alt="",
+                width="243",
+                height="180",
+                hi_def=True,
+                loading="lazy"
+              ) | safe
+            }}  
+          </div>
+        </div>
+        <div class="col-3 col-medium-3">
+          <h3>Smart city</h3>
+          <p>Harness the power of Ubuntu Core to keep distributed devices connected and secure.</p>
+          <p><a href="/internet-of-things/smart-city">Learn more&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+    </div>
   </div>
 </section>
 
 <section class="p-strip--light is-bordered">
   <div class="row">
     <div class="col-8">
       <h2>IoT App Store</h2>
-      <p>A private, custom, dedicated app store. Enterprise-ready for efficient software distribution. Utilise our huge range of existing apps or build your own. Included in the SMART START package.</p>
+      <p>A custom, private app store. Enterprise-ready for efficient software distribution and IoT package management, with on-premise options to support your use case. Utilise our huge range of existing apps or build your own. Included in the SMART START package.</p>
     </div>
+    <p><a href="https://assets.ubuntu.com/v1/d6d1d3fc-IoT+App+Store+Datasheet+v3.pdf">Read the datasheet&nbsp;&rsaquo;</a></p>
+    <p><a href="/internet-of-things/appstore" class="p-button--positive">Learn more</a></p>
   </div>
   <div class="u-fixed-width u-hide--small">
-    {{
-      image(
-        url="https://assets.ubuntu.com/v1/43b992df-private-app-store.png",
-        alt="Snap store screenshot",
-        width="1040",
-        height="510",
+    {{ 
+      image (
+        url="https://assets.ubuntu.com/v1/fbde412c-private-app-store.png",
+        alt="",
```

Comment:
👍 for not removing the alt tag. Adding `role="presentation"` on top of it would be even better (wider support, not all browsers/screen readers interpret the empty alt tag in the same way).